nsot/api/views.py

```diff
@@ -664,6 +644,28 @@ def reserved(self, request, site_pk=None, *args, **kwargs):
         objects = models.Network.objects.reserved()
         return self.list(request, queryset=objects, *args, **kwargs)
 
+    # Shoutout to jathanism for suggesting the move of perform_destroy(()
+    # to the NetworkViewSet to make it more explicit.
+    def perform_destroy(self, instance):
+        """
+        Overload default to handle non-serializer exceptions, and log Change
+        events.
+        :param instance:
+            Model instance to delete
+        """
+        log.debug('NsotViewSet.perform_destroy() obj = %r', instance)
```

Comment:
It looks like this log message didn't get updated when it was moved from `NsotViewSet`

Reply:
Nice catch!